proxysql/datadog_checks/proxysql/proxysql.py

```diff
@@ -0,0 +1,294 @@
+from collections import defaultdict
+from contextlib import closing, contextmanager
+
+import pymysql
+import pymysql.cursors
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.errors import ConfigurationError
+
+GAUGE = "gauge"
+RATE = "rate"
+COUNT = "count"
+MONOTONIC = "monotonic_count"
+
+STATS_MYSQL_GLOBAL = {
+    "Active_Transactions": ("proxysql.active_transactions", GAUGE),
+    "Query_Processor_time_nsec": ("proxysql.query_processor_time_nsec", RATE),
+    "Questions": ("proxysql.questions", RATE),
+    "Slow_queries": ("proxysql.slow_queries", RATE),
```

Comment:
```suggestion
    "Slow_queries": ("proxysql.slow_queries", COUNT),
```
I think this may be more appropriate as count or gauge, depending on if slow_queries is always increasing.

Reply:
Same doubt as with Questions